csrc/executor_kernel_arg.h

```diff
@@ -28,9 +28,6 @@ namespace nvfuser {
 //! compilation, we are not unnecessarily holding memory that is not needed.
 class TORCH_CUDA_CU_API KernelArgumentHolder {
  public:
-  //! create KernelArgumentHolder from c10 inputs. Note that we we not taking
-  //! the ownership of the memory from the original inputs, but just recording
-  //! its meta data for kernel execution/compilation.
```

Comment:
Why is this deleted?

Reply:
I think this is no longer true. We are taking the ownership, because we keep an instance of `at::Tensor`.